subsys/nrf_security/src/drivers/cracen/cracenpsa/src/platform_keys/platform_keys.c

```diff
@@ -400,11 +400,6 @@ psa_status_t cracen_platform_get_builtin_key(psa_drv_slot_number_t slot_number,
 			psa_set_key_algorithm(attributes, PSA_ALG_PURE_EDDSA);
 			psa_set_key_usage_flags(attributes, PSA_KEY_USAGE_VERIFY_MESSAGE);
 		} else if (key.sicr.type == PSA_KEY_TYPE_AES) {
-			/* Currently AES-KW is not supported, so the key stored in SICR
```

Comment:
Nit, but this doesn't say that they keys are stored as plaintext, it says that the key material is used directly without AES key wrapping or a similar mechanism.  But I am ok to remove completely as well since the usage of these keys might change soon.

Reply:
Isn't that what AES key wrapping is?

That you use AES to encrypt a key so it isn't stored in plaintext?

Reply:
Thank you for the clarification, could we also make it clear in the comment that the key stored in SICR is stored encrypted?

i.e.

Currently AES-KW is not supported, so the key stored **encrypted** in SICR is always used as the content encryption key to decrypt the firmware (using AES-GCM). When AES-KW is supported, this will possibly need to be extended so that both AES-KW (key stored in SICR used as KEK) and direct decryption (**encrypted** key stored in SICR used as CEK) can be supported.

Reply:
Sure, sounds good!

Reply:
@ahasztag : pushed